.ruby-version

```diff
@@ -1 +1 @@
-2.2.3
+2.3.0
```

Comment:
I think we can remove this actually. Testing on multiple ruby versions should be up to Travis for an OSS gem. We're not a rails application, just an engine. I don't think Administrate should even be aware of the ruby version.